front/app/containers/Admin/pagesAndMenu/EditHomepage/HeroBanner/index.tsx

```diff
@@ -192,7 +192,6 @@ const EditHomepageHeroBannerForm = () => {
                 localSettings.banner_signed_out_header_overlay_opacity
               }
               headerBg={localSettings.header_bg}
-              setFormStatus={setFormStatus}
```

Comment:
Was not really used, only as a dependency in useEffect, but it was not used in useEffect.